src/d3a/d3a_core/simulation.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig =
         self._load_setup_module()
         self._init(**self.initial_params)
         self._init_events()
+        validate_const_settings_for_simulation()
```

Comment:
I assume you want to validate the simulation const settings after loading the setup.py file ? This is nice.

Reply:
Yes, I should have added a comment in the PR.